### DISCUSSION TITLE
MAIANA demo set for a YouTube tutorial

### DISCUSSION THREAD

**Discussion by SirReal-surreal** [peterantypas/maiana#140] [General]
Hi there!
I got a MacArthur HAT because I was asked to create to YouTube tutorials helping with setup, functionalities and integration. I did the first two, a Signal K special will follow today and I also covered dAISy HAT integration.
However, I would love to cover also the MAIANA module. But I can't get it anywhere currently.
As I am doing these vids now (summer time is sailing time) I try finding a sample. 
I do these vids to help others, spending much much more as I earn with it and I did reactivate my channel just because the MacArthur HAT is a great chance for the community to create own solutions, what is what I love.

So, if anyone can help me out with a MAIANA demo case source, would be great.

MacArthur first tests on Node-RED: https://youtu.be/PnzAWgP4ipQ
MacArthur with OpenPlotter: https://youtu.be/PCKWElaCTPg
Signal K special (will be published today): https://youtu.be/UoXCQvN6W0c
Signal K API/WS special: In 14 days
MAIANA special: ??????

Cheers!
Malte

**Comment by muebau** [comment]
If you are unable to obtain one yourself, you may wish to look at the one we have installed, as we are not too far apart geographically.

**Reply by SirReal-surreal** [reply]
Thanks - cool. I will first try to get one, but if not successful I might come back to your kind offer!

**Comment by peterantypas** [comment]
Please send an email to maiana.kits at gmail dot com

**Reply by SirReal-surreal** [reply]
Thanks Peter - done!
